packages/valko-ui-components/src/types/Timepicker.ts

```diff
@@ -1,7 +1,7 @@
-import type { DefaultComponent } from './common'
+import type { DefaultComponent, AriaAttributes } from './common'
 import type { TimeProps } from './Time'
 
-export interface TimepickerProps extends DefaultComponent, TimeProps {
+export interface TimepickerProps extends DefaultComponent, TimeProps, Pick<AriaAttributes, 'aria-label' | 'aria-labelledby' | 'aria-describedby' | 'aria-invalid' | 'aria-required'> {
```

Comment:
Check if this is shorter using Omit instead of Pick, since you are picking a lot of props